backend/src/main/java/com/bakdata/conquery/models/datasets/Table.java

```diff
@@ -43,6 +48,10 @@ public class Table extends Labeled<TableId> implements NamespacedIdentifiable<Ta
 	@JsonManagedReference
 	private Column primaryColumn;
 
+	@JsonIgnore
+	@Getter(AccessLevel.NONE)
+	private Object2IntMap<Column> columnPositions = new Object2IntOpenHashMap<>();
```

Comment:
Ich hatte es dir im vorherigen PR schon angemerkt, du kannst hier keine Map benutzen, das wird die Performance komplett tanken. Das muss in der Column als Feld gecached sein.

Reply:
Ah danke nochmal für den hinweis, dann dreh ich es um und mache einen kommentar dran

Reply:
Ich habe es jetzt umgebaut:

Eine Table wird nach der Deserializierung automatisch initialisiert, dabei werden die positions in den columns gesetzt.